### PR TITLE
Implementation of PR that I created on the API

### PR DESCRIPTION
**Pull request by irrld**
https://github.com/CloudburstMC/API/pull/13

### CODE REVIEW

File: src/main/java/org/cloudburstmc/server/CloudServer.java [hunk 1]
```diff
@@ -1827,11 +1832,11 @@ public StorageRegistry getStorageRegistry() {
         return storageRegistry;
     }
 
-    public CloudGameRuleRegistry getGameRuleRegistry() {
+    public GameRuleRegistry getGameRuleRegistry() {
```
Comment (SupremeMortal) [inline src/main/java/org/cloudburstmc/server/CloudServer.java]: These intentionally used the `Cloud` derivative to make it easier for people who want to use internal APIs.

**Review by SupremeMortal — CHANGES_REQUESTED** [review]

File: src/main/java/org/cloudburstmc/server/CloudServer.java (hunk 1 above)
Comment (irrld) [reply src/main/java/org/cloudburstmc/server/CloudServer.java:1835]: They can cast if they want, can't they? If we provide the internal implementation here we have to remove it from the Server interface [here](https://github.com/CloudburstMC/API/pull/13/commits/7eeb4f550d77d7f87a365efb9c79bc6af885d8c1#diff-9a226795261e9f581c3a23652f65343e6c5df40a47ce0691ce09924850904a6eR40)

I can remove it if you want me to.

Comment (irrld) [reply src/main/java/org/cloudburstmc/server/CloudServer.java:1835]: OH actually we don't need to remove from the interface sorry lol.

Comment (irrld) [reply src/main/java/org/cloudburstmc/server/CloudServer.java]: Maybe we can make others return the `Cloud` too? What do you think about that. 
I mean ItemRegistry and RecipeRegistry.

**Pull request closed without merging**